java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture7/tasks/_2/CalculateAtomicObjects.java

```diff
@@ -0,0 +1,57 @@
+package com.zoolatech.lecture7.tasks._2;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class CalculateAtomicObjects implements Operations {
+    private final AtomicInteger getValue;
+    private int value;
+
+    public CalculateAtomicObjects(int value, AtomicInteger getValue) {
+        this.value = value;
+        this.getValue = getValue;
+    }
+
+    public AtomicInteger currentValueForAtomicInteger() {
+        return getValue;
+    }
+
+    @Override
+    public int addition(int integer) {
+        value = integer;
+        return getValue.addAndGet(value);
+    }
+
+    @Override
+    public int subtraction(int integer) {
+        value = integer;
+        return getValue.get() - value;
```

Comment:
the method doesn't change the value stored in `getValue` (compare it with the method `addition`)